scripts/performance/blas/alltime.py

```diff
@@ -0,0 +1,786 @@
+#!/usr/bin/python3
+
+import sys, getopt
+sys.path.append('../../../clients/common/')
```

Comment:
Note this path is hardcoded to reuse yaml scripts that are used by the client. If anyone has a suggestion for a better way to do this please let me know.

Reply:
I would think you could just keep a relative path and append to the current working directory to prefix when fetching yaml by file name.   Adding to the system path could have side effects down the road, but probably fine for now.   May need an environment variable for the full path if this python is invoked from different locations in CI and standalone.